### PR TITLE
Switch to using manageiq-style for linting dependencies

### PR DESCRIPTION
**Pull request by bdunne**
- [ ] https://github.com/ManageIQ/amazon_ssa_support/pull/50
- [ ] https://github.com/ManageIQ/dbus_api_service/pull/19
- [ ] https://github.com/ManageIQ/httpd_configmap_generator/pull/50
- [ ] https://github.com/ManageIQ/log_decorator/pull/12
- [ ] https://github.com/ManageIQ/manageiq-api/pull/845
- [ ] https://github.com/ManageIQ/manageiq-appliance/pull/286
- [ ] https://github.com/ManageIQ/manageiq-appliance-build/pull/419
- [ ] https://github.com/ManageIQ/manageiq-appliance_console/pull/127
- [ ] https://github.com/ManageIQ/manageiq-automation_engine/pull/458
- [ ] https://github.com/ManageIQ/manageiq-consumption/pull/180
- [ ] https://github.com/ManageIQ/manageiq-content/pull/669
- [ ] https://github.com/ManageIQ/manageiq-decorators/pull/36
- [ ] https://github.com/ManageIQ/manageiq-gems-pending/pull/491
- [ ] https://github.com/ManageIQ/manageiq-graphql/pull/86
- [ ] https://github.com/ManageIQ/manageiq-messaging/pull/49
- [ ] https://github.com/ManageIQ/manageiq-providers-amazon/pull/625
- [ ] https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/222
- [ ] https://github.com/ManageIQ/manageiq-providers-autosde/pull/34
- [ ] https://github.com/ManageIQ/manageiq-providers-azure/pull/396
- [ ] https://github.com/ManageIQ/manageiq-providers-azure_stack/pull/41
- [ ] https://github.com/ManageIQ/manageiq-providers-foreman/pull/64
- [ ] https://github.com/ManageIQ/manageiq-providers-google/pull/160
- [ ] https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/89
- [ ] https://github.com/ManageIQ/manageiq-providers-ibm_terraform/pull/9
- [ ] https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/379
- [ ] https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/172
- [ ] https://github.com/ManageIQ/manageiq-providers-lenovo/pull/303
- [ ] https://github.com/ManageIQ/manageiq-providers-nsxt/pull/21
- [ ] https://github.com/ManageIQ/manageiq-providers-nuage/pull/228
- [ ] https://github.com/ManageIQ/manageiq-providers-openshift/pull/181
- [ ] https://github.com/ManageIQ/manageiq-providers-openstack/pull/588
- [ ] https://github.com/ManageIQ/manageiq-providers-ovirt/pull/495
- [ ] https://github.com/ManageIQ/manageiq-providers-redfish/pull/125
- [ ] https://github.com/ManageIQ/manageiq-providers-scvmm/pull/167
- [ ] https://github.com/ManageIQ/manageiq-providers-vmware/pull/581
- [ ] https://github.com/ManageIQ/manageiq-rpm_build/pull/50
- [ ] https://github.com/ManageIQ/manageiq-schema/pull/488
- [ ] https://github.com/ManageIQ/manageiq-smartstate/pull/143
- [ ] https://github.com/ManageIQ/manageiq-ui-classic/pull/7073
- [ ] https://github.com/ManageIQ/manageiq/pull/20185
- [ ] https://github.com/ManageIQ/manageiq-v2v/pull/1145
- [ ] https://github.com/ManageIQ/activerecord-id_regions/pull/16
- [ ] https://github.com/ansible/ansible_tower_client_ruby/pull/142
- [ ] https://github.com/ManageIQ/azure-armrest/pull/396
- [ ] https://github.com/ManageIQ/linux_admin/pull/219

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,11 +44,8 @@ gem 'travis',         '~> 1.7.6'
 gem 'awesome_spawn',        '>= 1.4.1'
 gem 'default_value_for',    '>= 3.1.0'
 gem 'haml',                 '~> 5.1',    :require => false # force newer version of haml
-gem 'haml_lint',            '~> 0.35.0', :require => false
+gem 'manageiq-style',       '~> 1',      :require => false
```
Comment (Fryguy) [inline Gemfile:47]: can we leave it open ended?

**Comment by Fryguy** [comment]
@bdunne I know you're still in the middle of this, but please also change the plugin generator to include these changes.

File: Gemfile.lock [hunk 2]
```diff
@@ -420,20 +425,17 @@ DEPENDENCIES
   faraday-http-cache (~> 2.0.0)
   foreman (~> 0.64.0)
   haml (~> 5.1)
-  haml_lint (~> 0.35.0)
   influxdb (~> 0.3.13)
   jquery-rails
   listen
+  manageiq-style (~> 1)
```
Comment (Fryguy) [inline Gemfile.lock:431]: Think you need to rebuild the Gemfile.lock now that you changed the Gemfile

File: Gemfile (hunk 1 above)
Comment (bdunne) [reply Gemfile]: Yep, done

**Comment by kbrock** [comment]
fun stuff. thank you brandon

**Pull request merged**